torchaudio/backend/common.py

```diff
@@ -135,6 +161,10 @@ def __init__(self,
 
 _SAVE_DOCSTRING = r"""Saves a Tensor on file as an audio file
 
+Warnings:
```

Comment:
I think this is unnecessary, because it's duplicated with the warning statement and because we don't do this in any other library.